src/app/templates/app/submit_new_license_base.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
```

Comment:
@BassCoder2808 
Having a new base file for submit license is not needed, this should be fixed in the base.html itself. 

Reasons:-
1. The whole website should have only one base file.
2. Maintenance is difficult if we have the same content in multiple files.
3. Website would be loading a new html page specifically for submit license page - which is not required.


Let's try to figure out how to fix this globally in base.html

Reply:
Sure @rtgdk , I will try to figure this out and will let you know once thats done